includes/notes/class-wc-payments-notes-loan-approved.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Display a notice to merchants that have WCPay installed to inform them
+ * that a loan offer has been approved from Stripe.
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+use Automattic\WooCommerce\Admin\Notes\Notes;
+use Automattic\WooCommerce\Admin\Notes\Note;
+use Automattic\WooCommerce\Admin\Notes\NoteTraits;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class WC_Payments_Notes_Additional_Payment_Methods
+ */
+class WC_Payments_Notes_Loan_Approved {
+	use NoteTraits;
+
+	/**
+	 * Prefix of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-payments-notes-loan-approved';
+
+	/**
+	 * Nonce action name
+	 */
+	const NOTE_ACTION = 'view-capital-page';
+
+	/**
+	 * Loan information to build the message.
+	 *
+	 * @var array
+	 */
+	private static $loan_info;
+
+	/**
+	 * Get the note.
+	 */
+	public static function get_note() {
+		$note_class = WC_Payment_Woo_Compat_Utils::get_note_class();
+		$note       = new $note_class();
+
+		$note->set_title( __( 'Your capital loan has been approved!', 'woocommerce-payments' ) );
+		$note->set_content(
+			sprintf(
+				// Translators: %1: total amount lent to the merchant formatted in the account currency.
+				__(
+					'Congratulations! Your capital loan has been approved and %1$s was deposited in to the bank account linked to WooCommerce Payments. You\'ll automatically repay the loan, plus a flat fee, through a fixed percentage of each WooCommerce Payments transaction.',
+					'woocommerce-payments'
+				),
+				WC_Payments_Explicit_Price_Formatter::get_explicit_price( wc_price( self::$loan_info['details']['advance_amount'] / 100 ) )
```

Comment:
I get that right now we're only opening Capital to US, but it doesn't hurt to leave the code prepared for a possible future expansion. This (untested) change should take into account other currencies, including zero-decimal currencies:

```suggestion
				$dummy_order = wc_create_order();
				$dummy_order->set_currency( self::$loan_info['details']['currency'] );
				WC_Payments_Explicit_Price_Formatter::get_explicit_price( wc_price( WC_Payments_Utils::interpret_stripe_amount( self::$loan_info['details']['advance_amount'], $dummy_order ) ) )
```

This code assumes that you'll pass (and store) the loan currency on the note. `get_explicit_price` only accepts an `WC_Order` object so that makes the code a bit weird (if you change it to accept an optional `$currency` string it would be simpler).